docs/guides/podman.md

```diff
@@ -0,0 +1,162 @@
+# Podman
+[Podman](https://podman.io/) is an alternative to docker; 
+it's a daemonless container engine for developing, managing and running OCI containers on a Linux system.
+Podman also works on Mac using a [podman machine](https://docs.podman.io/en/latest/markdown/podman-machine.1.html).
+
+## Prerequisites
+ - [Install podman](https://podman.io/getting-started/installation)
+ - Mac: ensure a [podman machine](podman machine) is running.
+ - Linux: for [multi-platform builds](https://docs.earthly.dev/docs/guides/multi-platform), install [qemu-user-static](https://github.com/multiarch/qemu-user-static).
+ - [WITH DOCKER](https://docs.earthly.dev/docs/earthfile#with-docker) requires rootful mode.
+   - Linux: run with `sudo` (i.e., `sudo earthly -P +with-docker-target`)
+   - Mac: run a [rootful machine](https://docs.podman.io/en/latest/markdown/podman-machine-set.1.html#rootful).
+
+## Getting started
+When earthly starts a check is done to determine what frontend is available.
+By default, earthly will attempt to use docker and then fall back to podman.
+If you wish to change the behavior of the startup check, run the following command:
+
+```bash
+# Configure earthly to use podman
+> earthly config global.container_frontend podman-shell
+
+# Configure earthly to use docker
+> earthly config global.container_frontend docker-shell
```

Comment:
```suggestion
earthly config global.container_frontend podman-shell

# Configure earthly to use docker
earthly config global.container_frontend docker-shell
```

We've used this convention on other pages: no terminal prompt, to make it easier to copy-paste.